db/migrate/20230718082519_add_primary_key_to_backend_events.rb

```diff
@@ -0,0 +1,25 @@
+class AddPrimaryKeyToBackendEvents < ActiveRecord::Migration[5.2]
+  def up
+    remove_index :backend_events, :id
+
+    if System::Database.mysql?
+      execute 'ALTER TABLE backend_events ADD PRIMARY KEY (id)'
+    elsif System::Database.postgres?
+      execute 'ALTER TABLE backend_events ADD CONSTRAINT backend_events_pk PRIMARY KEY (id)'
+    else
+      execute 'ALTER TABLE BACKEND_EVENTS MODIFY ID PRIMARY KEY'
+    end
+  end
+
+  def down
+    if System::Database.mysql?
+      execute 'ALTER TABLE backend_events DROP PRIMARY KEY'
+    elsif System::Database.postgres?
+      execute 'ALTER TABLE backend_events DROP CONSTRAINT backend_events_pk'
+    else
+      execute 'ALTER TABLE BACKEND_EVENTS DROP PRIMARY KEY'
+    end
+
+    add_index :backend_events, :id, unique: true
+  end
```

Comment:
As last time, I don't think we should provide `down` here. It would be very unsafe to revert. And revert will only succeed if migration already succeeded. At which point we don't want to allow going back.

Reply:
Reverting is highly convenient for development. I'll remove the down method just before merging when I'm sure I'm not making more changes to the migrations.